<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUpdatePosition</class>
 <widget class="QWidget" name="CUpdatePosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===============================
   Main Widget Background
   =============================== */
QWidget {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #f2f4f7,
        stop:1 #d9dee4
    );
    font-family: Segoe UI, Roboto, sans-serif;
    font-size: 14px;
    color: #1E1E1E;
}

/* ===============================
   Labels
   =============================== */
QLabel {
    color: #2c3e50;
    font-size: 14px;
    font-weight: 800;
}

/* ===============================
   Line Edits
   =============================== */
QLineEdit {
    background: #ffffff;
    border: 1px solid #b6bec7;
    padding: 6px 8px;
    border-radius: 6px;
    selection-background-color: #0078d7;
    selection-color: white;
}

QLineEdit:focus {
    border: 1px solid #0078d7;
    background: #fefefe;
}

/* ===============================
   Buttons
   =============================== */
QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #3478f6,
                                stop:0.5 #2d6bdd,
                                stop:1 #255bc0);
    color: white;
    padding: 8px 14px;
    border-radius: 8px;
    border: 2px solid rgba(52, 120, 246, 0.3);
    font-weight: 700;
    letter-spacing: 0.5px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #4e91ff,
                                stop:0.5 #3478f6,
                                stop:1 #2d6bdd);
    border: 2px solid rgba(52, 120, 246, 0.5);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #1d4ea3,
                                stop:1 #143970);
    border: 2px solid rgba(20, 57, 112, 0.8);
    padding: 9px 14px 7px 14px;
}

QPushButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #BDBDBD,
                                stop:1 #9E9E9E);
    border: 2px solid rgba(189, 189, 189, 0.3);
    color: rgba(255, 255, 255, 0.5);
}

/* Close Button (Optional Different Color) */
QPushButton#closeButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #e74c3c,
                                stop:0.5 #cf4436,
                                stop:1 #b83c30);
    border: 2px solid rgba(231, 76, 60, 0.3);
}

QPushButton#closeButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #f55f4e,
                                stop:0.5 #e74c3c,
                                stop:1 #cf4436);
    border: 2px solid rgba(231, 76, 60, 0.5);
}

QPushButton#closeButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #a53526,
                                stop:1 #7a221a);
    border: 2px solid rgba(165, 53, 38, 0.8);
    padding: 9px 14px 7px 14px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background:none;</string>
     </property>
     <property name="text">
      <string>LATITUDE :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_latitude"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">background:none;</string>
     </property>
     <property name="text">
      <string>LONGITUDE :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_longitude"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background:none;</string>
     </property>
     <property name="text">
      <string>ALTITUDE :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_altitude"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{background:none;}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_update">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
